<launch>
  <node name="kernelNode"          pkg="quadcopter_ctrl" type="kernelNode" args="5" output="screen"/>
  <node name="quadcopterRosCtrl_0" pkg="quadcopter_ctrl" type="quadcopterRosCtrl" args="0" output="screen"/>
  <node name="quadcopterRosCtrl_1" pkg="quadcopter_ctrl" type="quadcopterRosCtrl" args="1" output="screen"/> 
  <node name="quadcopterRosCtrl_2" pkg="quadcopter_ctrl" type="quadcopterRosCtrl" args="2" output="screen"/>
  <node name="quadcopterRosCtrl_3" pkg="quadcopter_ctrl" type="quadcopterRosCtrl" args="3" output="screen"/> 
  <node name="quadcopterRosCtrl_4" pkg="quadcopter_ctrl" type="quadcopterRosCtrl" args="4" output="screen"/> 
</launch>
